src/App.js

```diff
@@ -1,10 +1,15 @@
-import React from 'react'
+import React , { useEffect } from 'react'
+import Notes from './components/Notes';
 
 function App() {
+  useEffect(() => {
+    <Notes/>
```

Comment:
what you are trying to do here? nothing will happen here with this piece of code.